packages/components/src/components/ThemeProvider/ThemeProvider.tsx

```diff
@@ -12,30 +14,69 @@ type ThemeProviderProps = {
      * The theme to apply
      * @default "light"
      * */
-    theme: AvailableTheme;
+    theme?: AvailableTheme;
     /**
      * Direction to apply to the theme
      * @default "ltr"
      */
-    dir: 'ltr' | 'rtl';
+    dir?: 'ltr' | 'rtl';
+    /**
+     * Translations object to use for component strings.
+     * Import from '@frontify/fondue/components/locales' or provide your own.
+     * Must be a complete Translations object.
+     * @default English translations
```

Comment:
```suggestion
     * @default "en"
```

Reply:
Well, "en" is not really true... There is no string here.